File: leetcode/2265-partition-array-according-to-given-pivot/solution.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        g = []
        l = []
        e = []
        for i in nums:
            if i < pivot:
                l.append(i)
            elif i>pivot:
                g.append(i)
            else:
                e.append(i)
        return l+e+g

        

File: leetcode/2265-partition-array-according-to-given-pivot/solution.js
/**
 * @param {number[]} nums
 * @param {number} pivot
 * @return {number[]}
 */
var pivotArray = function(nums, pivot) {
    let pos =0
    for(let i=0;i<nums.length;i++){
        if(nums[i]<pivot){
            let temp = nums[i]
            for(let j=i;j>pos;j--){
                nums[j]=nums[j-1]
            }
            nums[pos]=temp
            pos++
        }
    }
    for(let i=0;i<nums.length;i++){
        if(nums[i] === pivot){
            let temp = nums[i]
            for(let j=i;j>pos;j--){
                nums[j] = nums[j-1]
            }
            nums[pos] = temp
            pos++
        }
    }
    return nums
};
